test/parallel/test-unicode-node-options.js

```diff
@@ -0,0 +1,23 @@
+'use strict';
+// Flags: --expose-internals
+require('../common');
+const { getOptionValue } = require('internal/options');
+const assert = require('assert');
+const cp = require('child_process');
+
+const expected_redirect_value = 'foó';
+
+if (process.argv.length === 2) {
+  const NODE_OPTIONS = `--redirect-warnings=${expected_redirect_value}`;
+  const result = cp.spawnSync(process.argv0,
+                              ['--expose-internals', __filename, 'test'],
+                              {
+                                env: { NODE_OPTIONS },
```

Comment:
Does this help with the current CI failure?

```suggestion
                                env: { ...process.env, NODE_OPTIONS },
```

Reply:
Yes, it helped. Thanks!